admin-perspective/admin-perspective.md

```diff
@@ -0,0 +1,100 @@
+## OpenShift 4.4+ Designs
+Access design documentation for features slotted in the 4.4 release of OpenShift. Click links below for specific section details.
+
+- [Bulk actions](http://openshift.github.io/openshift-origin-design/admin-perspective/future-openshift/bulk-actions/bulk-actions)
+- [Cron Jobs](http://openshift.github.io/openshift-origin-design/admin-perspective/future-openshift/cron-jobs/cron-jobs)
+- [Diagnostics](http://openshift.github.io/openshift-origin-design/admin-perspective/future-openshift/diagnostics/diagnostics)
+- [Favoriting](http://openshift.github.io/openshift-origin-design/admin-perspective/future-openshift/favoriting/favoriting)
+- [Health Checks](http://openshift.github.io/openshift-origin-design/admin-perspective/future-openshift/health-checks/health-checks)
+- [Search page](http://openshift.github.io/openshift-origin-design/admin-perspective/future-openshift/search/search)
+- [Toolbars](http://openshift.github.io/openshift-origin-design/admin-perspective/future-openshift/toolbars/toolbars)
+
+## OpenShift 4.3 Designs
+Access design documentation for features slotted in the 4.3 release of OpenShift. Click links below for specific section details.
+
+- [Alertmanager config](http://openshift.github.io/openshift-origin-design/admin-perspective/monitoring/alertmanager-config/alertmanager-config)
+- [Chargeback](http://openshift.github.io/openshift-origin-design/future-openshift/admin-perspective/chargeback/chargeback)
+- [Custom Catalogs](http://openshift.github.io/openshift-origin-design/admin-perspective/future-openshift/custom-catalogs/custom-catalogs)
+- [Dashboards](http://openshift.github.io/openshift-origin-design/admin-perspective/dashboards/4.3/4.3-dashboards)
+
+## OpenShift 4.2 Designs
+Access design documentation for features slotted in the 4.2 release of OpenShift. Click links below for specific section details.
+
+- [API Explorer](http://openshift.github.io/openshift-origin-design/future-openshift/admin-perspective/api-explorer/api-explorer)
+- [Command Line Tools](http://openshift.github.io/openshift-origin-design/admin-perspective/future-openshift/command-line-tools/command-line-tools)
```

Comment:
```suggestion
- [Cluster Overview Dashboard](http://openshift.github.io/openshift-origin-design/admin-perspective/cluster-dashboard/cluster-dashboard)
- [Command Line Tools](http://openshift.github.io/openshift-origin-design/admin-perspective/future-openshift/command-line-tools/command-line-tools)
```

Reply:
ya I was wondering about this one too, should we put this inside the dashboard directory so it's `admin-perspective/dashboards/cluster-dashboard/`